webapp/pages/settings/index.vue

```diff
@@ -123,51 +121,38 @@ export default {
       clearTimeout(timeout)
       timeout = setTimeout(() => this.requestGeoData(value), 500)
     },
-    processCityResults(res) {
-      if (!res || !res.data || !res.data.features || !res.data.features.length) {
+    processLocationsResult(places) {
+      if (!places.length) {
         return []
       }
-      const output = []
-      res.data.features.forEach((item) => {
-        output.push({
-          label: item.place_name,
-          value: item.place_name,
-          id: item.id,
+      const result = []
+      places.forEach((place) => {
+        result.push({
+          label: place.place_name,
+          value: place.place_name,
+          id: place.id,
         })
       })
 
-      return output
+      return result
     },
-    requestGeoData(e) {
-      if (this.axiosSource) {
-        // cancel last request
-        this.axiosSource.cancel()
-      }
-
+    async requestGeoData(e) {
       const value = e.target ? e.target.value.trim() : ''
-      if (value === '' || value.length < 3) {
+      if (value === '') {
```

Comment:
Are you sure you want to start the query with just one char of input? I would prefer starting the query if we have at least 3 chars of input.

Reply:
For what reason?